Examples are not being updated on MDN

Some time between October 23 and October 26 it seems like changes made to content in interactive-examples stopped being picked up on MDN.

This change, merged on October 23: https://github.com/mdn/interactive-examples/pull/1193 is visible on the MDN page: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/normalize.

But this change, merged on October 26: https://github.com/mdn/interactive-examples/pull/1182 is not visible: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/entries.

And no subsequent changes are visible (e.g. https://github.com/mdn/interactive-examples/pull/1194, https://github.com/mdn/interactive-examples/pull/1204).

Perhaps related to the BoB split?

### Some background (for terminology)
Whenever the `prod` branch of this repo is modified, Jenkins will perform a `build` step followed by a `s3 sync` step. Here's the `build` step:
```sh
stage ('build') {
    try {
        sh 'bin/build.sh'
    } catch(err) {
        sh "bin/irc-notify.sh --stage 'build " + env.BRANCH_NAME + "' --status 'failed'"
        throw err
    }
}
```
which calls `bin/build.sh`:
```sh
#!/bin/bash -x
cd "$(dirname ${BASH_SOURCE[0]})"/..
echo "Starting build"
docker run -v "$PWD":/mdn -w /mdn node:latest bash -c "/usr/local/bin/npm install && /usr/local/bin/npm run build"
echo "Build finished"
```
### My discoveries (so far)
- the Jenkins `build` step (not the `s3 sync` step) has been failing, although silently, since [this commit](https://github.com/mdn/interactive-examples/commit/8210c312c54a392e27277b5906f8980077d0c87b) 14 days ago
- the part of the `build` step that has been failing is the `/usr/local/bin/npm install`
- it appears to have started failing due to leftover contents of the `./node_modules` directory from a previous install suddenly interfering with the current install, for example:
    ```sh
    npm ERR! path /mdn/node_modules/stylelint/node_modules/.bin/autoprefixer
    npm ERR! code EEXIST
    npm ERR! Refusing to delete /mdn/node_modules/stylelint/node_modules/.bin/autoprefixer: is outside /mdn/node_modules/stylelint/node_modules/autoprefixer and not a link
    npm ERR! File exists: /mdn/node_modules/stylelint/node_modules/.bin/autoprefixer
    npm ERR! Move it away, and try again.
    ```
- it's been failing silently because although the exit code from `bin/build.sh` is used to trigger the `try-catch` and report failures, that exit code will always be `0` (successful) since the exit code of the last command is used as the exit code of the script, and in this case that command is `echo "Build finished"`
- I re-played one of the earlier `prod` builds, but this time doing a `rm -rf ./node_modules` prior to calling `bin/build.sh`, and then the `npm install` step completed successfully
- when I re-played the latest `prod` build, which had failed originally, it also performed the `npm install` step successfully, even though I had not removed the leftover `node_modules` directory prior to the build, which I think is due to the fact that the `node_modules` directory had been modified by the earlier re-play such that its contents no longer conflicted with subsequent runs
- the interactive examples should currently be up-to-date on MDN

### Next steps
- fix the `bin/build.sh` script such that `npm install` or `npm run build` failures are reflected in its exit code
- find a reliable way to remove the `node_modules` directory prior to the `npm install` (it doesn't seem to be as simple as adding a `rm -rf ./node_modules` prior to calling `bin/build.sh`, since after the first time it ran successfully, it then started failing with permission-denied errors)

--- a/bin/build.sh
+++ b/bin/build.sh
@@ -1,5 +1,11 @@
 #!/bin/bash -x
 cd "$(dirname ${BASH_SOURCE[0]})"/..
 echo "Starting build"
-docker run -v "$PWD":/mdn -w /mdn node:latest bash -c "/usr/local/bin/npm install && /usr/local/bin/npm run build"
-echo "Build finished"
+docker run -v "$PWD":/mdn -w /mdn node:latest bash -c "rm -rf node_modules && /usr/local/bin/npm install && /usr/local/bin/npm run build"
+if [ $? -eq 0 ]; then
+    echo "Build finished"
+    exit 0
+else
+    echo "Build failed" >&2
+    exit 1
+fi

